.editorconfig

```
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
```

Maybe we could have the **indent style** set to **tabs**.
```suggestion
indent_style = tab
```
And now the `indent_size = 2` should let GitHub know to display TABs with **2 spaces instead of the default 8** spaces! 😎

Reference on [StackOverflow](https://stackoverflow.com/questions/8833953/how-to-change-tab-size-on-github#answer-33831598)